strings/french.strings

```diff
@@ -208,1285 +208,1285 @@
+"After the onboarding period has ended, everyone will be asked to vote on whether or not a new member should be added. But for now, enjoy 60 free passes!" = "Une fois la période d'initiation terminée, pour tout ajout ultérieur d'un nouveau membre il sera demandé à chaque membre actuel de voter. Mais pour l'heure, profitez de 60 pass gratuits !";
 
 /* MISSING TRANSLATION - views/containers/user-settings/AppLogs.vue */
-"If you encounter problems, please download the logs and {a_}send them to us{_a}." = "If you encounter problems, please download the logs and {a_}send them to us{_a}.";
+"If you encounter problems, please download the logs and {a_}send them to us{_a}." = "En cas de problème, veuillez télécharger le fichier journal et {a_}nous l'envoyer{_a}.";
 
-/* MISSING TRANSLATION - views/containers/access/LoginForm.vue, views/containers/access/SignupForm.vue, views/containers/group-settings/GroupLeaveModal.vue, views/containers/user-settings/UserProfile.vue */
-"Password" = "Password";
+/* views/containers/access/LoginForm.vue, views/containers/access/SignupForm.vue, views/containers/group-settings/GroupLeaveModal.vue, views/containers/user-settings/UserProfile.vue */
+"Password" = "Mot de passe";
 
 /* MISSING TRANSLATION - views/pages/DesignSystem.vue */
-"NB: at the moment we don't have sub sub modal, there for the implementation is flexible to allow an inifinty of modal but the animation is not implemented" = "NB: at the moment we don't have sub sub modal, there for the implementation is flexible to allow an inifinty of modal but the animation is not implemented";
+"NB: at the moment we don't have sub sub modal, there for the implementation is flexible to allow an inifinty of modal but the animation is not implemented" = "NB: pour le moment nous n'avons pas encore de dialogues sub-sub-modaux, et donc l'implémentation est suffisamment flexible pour permettre une infinité de dialogues modaux mais l'animation n'est pas implémentée";
 
-/* MISSING TRANSLATION - views/containers/contributions/PaymentMethods.vue */
-"Payment value" = "Payment value";
+/* views/containers/contributions/PaymentMethods.vue */
+"Payment value" = "Valeur du paiement";
 
-/* MISSING TRANSLATION - views/containers/payments/PaymentRowTodo.vue */
-"Cannot dismiss a payment that hasn't been sent yet." = "Cannot dismiss a payment that hasn't been sent yet.";
+/* views/containers/payments/PaymentRowTodo.vue */
+"Cannot dismiss a payment that hasn't been sent yet." = "Il n'est pas possible de décliner un paiement qui n'a pas encore été envoyé.";
 
-/* MISSING TRANSLATION - views/containers/proposals/ProposalItem.vue */
-"Remove {user} from the group." = "Remove {user} from the group.";
+/* views/containers/proposals/ProposalItem.vue */
+"Remove {user} from the group." = "Renvoyer {user} du groupe.";
```

Comment:
Exclure was my first idea, since it has the same meaning and is shorter.

Then I chose to use Renvoyer because it sounded slightly less harsh to me than Exclure, but I am changing that if you and/or @SGrondin can confirm it sounds good.